meinberlin/apps/contrib/templates/meinberlin_contrib/includes/proposal_list_item.html

```diff
@@ -25,21 +25,19 @@ <h3 class="list-item__title">
         </a>
     </h3>
     <div class="list-item__labels">
-        {% if object.category %}
-            <span class="label label--big">{{ object.category }}</span>
-        {% endif %}
-        {% if object.point_label %}
-            <span class="label label--big">
-                <i class="fas fa-map-marker-alt" aria-hidden="true"></i>
-                {{ object.point_label }}
+        {% for badge in object.item_badges_for_list  %}
```

Comment:
maybe wrapping this into a subtemplate and use for all other lists?

Reply:
Yeah, I replaced it everywhere to make it easier to see that it's all the same, when I use the idea_list_item for all the list items in the third commit. With using that same item everywhere, the item_badges_for_list are also only be used once in one template (but only after the third commit...)

Reply:
ah, just realized it! cool then its good from my side!